tools/docker-images/clp-package/build.sh

```diff
@@ -5,10 +5,25 @@ set -o pipefail
 
 script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
 repo_root=${script_dir}/../../../
+iid_file="${repo_root}/build/clp-package-image.id"
+
+# Remove the previous image to save space, but only after the build so its contents can be reused.
+prev_image_id=""
+if [[ -f "$iid_file" ]]; then
+    prev_image_id=$(cat "$iid_file")
+fi
+cleanup() {
```

Comment:
Can we move this function to the top of the file, just after the `set` commands? That makes it easier to read the control flow imo.